lang/en/ratingallocate.php

```diff
@@ -202,7 +202,9 @@
 $string['choice_title'] = 'Title';
 $string['choice_title_help'] = 'Title of the choice. *Attention* all active choices will be displayed while ordered by title.';
 $string['choice_usegroups'] = 'Restrict visibility by groups';
-$string['choice_usegroups_help'] = 'If selected, this group will only be visible to the members of the specified groups.';
+$string['choice_usegroups_help'] = '* If selected, this group will only be visible to the members of the specified groups.
```

Comment:
Shouldn't this be "this _choice_ will only be visible ..."?

Reply:
Yes, I think so. Would be nice if I corrected this error in the first place... ;-)

I amended the commit.